### PR TITLE
Update scss

### PR DESCRIPTION
**Pull request by EmmanuelDrouin**
- Add mobile variable to structure
- Update scss

### CODE REVIEW

File: packages/Theme/index.scss [hunk 1]
```diff
@@ -135,6 +119,16 @@ a {
   text-align: right;
 }
 
+.white {
```
Comment (antho1404) [inline packages/Theme/index.scss]: why these classes? white, gold, success, we never use them in the components

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready for review

File: packages/Theme/index.scss (hunk 1 above)
Comment (EmmanuelDrouin) [reply packages/Theme/index.scss]: We use these classes for the website, true they are useless here, i will remove these

Comment (EmmanuelDrouin) [reply packages/Theme/index.scss]: Done

**Pull request merged**